run-tests.sh

```diff
@@ -131,7 +131,7 @@ test_galaxy_internal() {
 	shift
 
 	docker run -d -p $GALAXY_PORT:80 --name $GALAXY_DOCKER_NAME -v $TMP_DIRECTORY:$TMP_DIRECTORY -v $SCRIPT_DIR:$SCRIPT_DIR $GALAXY_DOCKER && \
-	mvn clean verify -B -P$profile -Djdbc.url=$JDBC_URL -Dirida.it.rootdirectory=$TMP_DIRECTORY -Dtest.galaxy.url=$GALAXY_URL -Dtest.galaxy.invalid.url=$GALAXY_INVALID_URL -Dtest.galaxy.invalid.url2=$GALAXY_INVALID_URL2 -Dsequence.file.base.directory=$SEQUENCE_FILE_DIR -Dreference.file.base.directory=$REFERENCE_FILE_DIR -Doutput.file.base.directory=$OUTPUT_FILE_DIR -Djdbc.pool.maxWait=$DB_MAX_WAIT_MILLIS $@
+	mvn clean verify -B -P$profile -Dfile.processing.decompress=false -Djdbc.url=$JDBC_URL -Dirida.it.rootdirectory=$TMP_DIRECTORY -Dtest.galaxy.url=$GALAXY_URL -Dtest.galaxy.invalid.url=$GALAXY_INVALID_URL -Dtest.galaxy.invalid.url2=$GALAXY_INVALID_URL2 -Dsequence.file.base.directory=$SEQUENCE_FILE_DIR -Dreference.file.base.directory=$REFERENCE_FILE_DIR -Doutput.file.base.directory=$OUTPUT_FILE_DIR -Djdbc.pool.maxWait=$DB_MAX_WAIT_MILLIS $@
```

Comment:
Any issue leaving them compressed here and us not testing it uncompressed?  Or will similar functions get tested uncompressed in the pipeline testing?

Reply:
I've changed this behaviour in 64d721c474f2e542e31238caaa925134a7fd341a, 93a2b2a72651d56eeddf7902518a37f00f7183e3, and 6078af440582bfd55f11533acefed1df8ac5487f so that `file.processing.decompress=true` is always set.

In 93a2b2a72651d56eeddf7902518a37f00f7183e3 specifically I modify the `GzipFileProcessor` so that it can be dynamically enabled and disabled. That way, for specific tests, I can disable it, but it should always default to enabled.